agave_app/main.cpp

```diff
@@ -159,9 +160,14 @@ main(int argc, char* argv[])
 
     result = a.exec();
   } else {
-    agaveGui* w = new agaveGui();
-    w->show();
-    result = a.exec();
+    try {
+      agaveGui* w = new agaveGui();
+      w->show();
+      result = a.exec();
+
+    } catch (...) {
+      LOG_ERROR << "Exception caught in main";

```

Comment:
Just continue on the error in main?  I suppose the program likely is crashing now anyway.